lib/fizzy/stack.hpp

```diff
@@ -58,32 +58,54 @@ class Stack : public std::vector<T>
 
 class OperandStack
 {
+    /// The size of the pre-allocated internal storage: 128 bytes.
+    static constexpr auto small_storage_size = 128 / sizeof(uint64_t);
```

Comment:
Do we want to fine-tune this, e.g. benchmark it with 64, 256, 512? Or should we postpone that until we have a better set of inputs?

Reply:
I tuned it to current benchmark set - the minimal value that fits all benchmarks (and unit tests by coincidence) in small storage.